docs/configuration/v2.x/metrics/index.md

```diff
@@ -134,6 +134,7 @@ We also provide a simplified way to scrape the following Azure resources:
 - [Azure IoT Hub](iot-hub)
 - [Azure IoT Hub Device Provisioning Service (DPS)](iot-hub-device-provisioning-service)
 - [Azure Key Vault](key-vault)
+- [Azure Kubernetes Service](kubernetes)
```

Comment:
Can you also provide a link in the resource discovery docs please?

Reply:
Added in 4d45384a92ee3569b97734e6f47fd17015bf3aa9